core/src/main/java/org/springframework/security/authorization/AuthorizationDecision.java

```diff
@@ -18,10 +18,23 @@
 
 /**
  * @author Rob Winch
+ * @author Evgeniy Cheban
  * @since 5.0
  */
 public class AuthorizationDecision {
 
+	/**
+	 * Represents granted {@link AuthorizationDecision}.
+	 * @since 5.8
+	 */
+	public static final AuthorizationDecision GRANTED = new AuthorizationDecision(true);
```

Comment:
I'm still a little hesitant to add these constants. 

They feel similar to `public static final Authentication AUTHENTICATED;` and `public static final Authentication UNAUTHENTICATED;`, which I don't think we'd want to do.

Reply:
I've removed these constants.